uint32 	msg_counter
float32 centerOffset
float32 curvature
